src/util.js

```diff
@@ -41,6 +41,21 @@ Util.isIFrame = function() {
   }
 };
 
+// Get a unique, incrementing Id value for any object on the page.
+Util.getUniqueId = (function(prefix) {
+  var i = Math.floor(Math.random() * 999) + 100;
+  var pfx = prefix || '';
+  function inc(pfx) {
+    if (!pfx) {
+      pfx = '';
+    } else {
+      pfx += '-';
+    }
+    return pfx + i++;
+  }
+  return inc;
+})();
```

Comment:
I guess my objection is to the code style. This is overly clever. I would just track the current ID as a variable in the Util namespace.